api/src/pcapi/connectors/clickhouse/queries/count_bookings.py

```diff
@@ -0,0 +1,41 @@
+import pydantic.v1 as pydantic_v1
+
+from pcapi.connectors.clickhouse.queries.base import BaseQuery
+
+
+class CountBookings(pydantic_v1.BaseModel):
+    individual_bookings: int
+    collective_bookings: int
+
+    class Config:
+        extra = "forbid"
+        orm_mode = True
+
+
+class CountBookingsQuery(BaseQuery[CountBookings]):
+    @property
+    def model(self) -> type[CountBookings]:
+        return CountBookings
+
+    @property
+    def raw_query(self) -> str:
+        return """
+            SELECT
+            (
+                SELECT
+                    count(*) AS quantity
```

Comment:
Détail : j'ai l'impression que `quantity` ne sert à rien.
```suggestion
                    count(*)
```